packages/aws-rfdk/lib/deadline/lib/database-connection.ts

```diff
@@ -336,13 +351,20 @@ class MongoDbInstanceDatabaseConnection extends DatabaseConnection {
   private static readonly DB_CERT_LOCATION: string = '/opt/Thinkbox/certs/mongo_client.pfx';
 
   public readonly containerEnvironment: { [name: string]: string };
+  public readonly databaseConstruct?: Construct;
 
   constructor(protected readonly props: MongoDbInstanceConnectionOptions) {
     super();
     this.containerEnvironment = {
       DB_TLS_CLIENT_CERT_URI: props.clientCertificate.cert.secretArn,
       DB_TLS_CLIENT_CERT_PASSWORD_URI: props.clientCertificate.passphrase.secretArn,
     };
+
+    if (props.database instanceof MongoDbInstance) {
+      if (props.database.mongoDataVolume instanceof Volume) {
+        this.databaseConstruct = props.database.mongoDataVolume;
```

Comment:
How have we tested that the `mongoDataVolume` is the correct Construct to track and not the `server`? Have we tried deploying a farm with a MongoDB and Secrets Management enabled?

Also I don't see any unit tests for these changes similar to the updated and added DocDB tests.